jupyter-extension/src/plugins/mount/components/Config/__tests__/Config.test.tsx

```diff
@@ -308,37 +331,49 @@ describe('config screen', () => {
       const submit = getByTestId('Config__pachdAddressSubmit');
 
       userEvent.type(input, 'grpc://test.com:31400');
-      submit.click();
+      await act(async () => {
```

Comment:
Why the need for all these `act()` calls now? Is it because we updated our react dependency?

Reply:
Yes, I am forgetting if it was react specifically or if it was one of the testing framework dependencies (probably the latter) but one of those two requires the click to be wrapped now.